dnf-behave-tests/dnf/api/transaction-remove-install.feature

```diff
@@ -0,0 +1,27 @@
+@dnf5
+@not.with_dnf=4
+Feature: transaction: remove a package with dependency, then install the dependency again
+
+
+Scenario: Construct query, remove vagare, then install labirinto
+Given I use repository "simple-base"
+Given I successfully execute dnf with args "install vagare"
+# vagare depends on labirinto
+ When I execute python libdnf5 api script with setup
+      """
+      goal = libdnf5.base.Goal(base)
+      goal.add_rpm_remove("vagare")
+      goal.add_rpm_install("labirinto")
+      execute_transaction(goal, "remove and install")
+      """
+   Then the exit code is 0
+    And stdout is
+ """
+ vagare-1.0-1.fc29.x86_64 : Remove
+ """
+ When I execute rpm with args "-q vagare"
+   Then the exit code is 1
+   And stdout contains "package vagare is not installed"
+ When I execute rpm with args "-q labirinto"
+   Then the exit code is 0
```

Comment:
Could you align the steps more uniformly?
Usually we have all the names of steps (after the `When`/`Given`... keywords) staring at the same column:
```Gherkin
Given I successfully execute dnf with args "install vagare"
# vagare depends on labirinto
 When I execute python libdnf5 api script with setup
      """
      goal = libdnf5.base.Goal(base)
      goal.add_rpm_remove("vagare")
      goal.add_rpm_install("labirinto")
      execute_transaction(goal, "remove and install")
      """
 Then the exit code is 0
  And stdout is
      """
      vagare-1.0-1.fc29.x86_64 : Remove
      """
```
(You can also check other existing features.)

Reply:
Oops, sorry, I've edited the files on a testing machine with weirdly set tabs and vim and I haven't checked them after scp.